Tests/test_file_xvthumb.py

```diff
@@ -1,13 +1,35 @@
-from helper import unittest, PillowTestCase
+from helper import hopper, unittest, PillowTestCase
 
-from PIL import XVThumbImagePlugin
+from PIL import Image, XVThumbImagePlugin
+
+TEST_FILE = "Tests/images/hopper.p7"
 
 
 class TestFileXVThumb(PillowTestCase):
 
+    def test_open(self):
+        # Act
+        im = Image.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "XVThumb")
+        self.assert_image_similar(im, hopper("P"), 49)
```

Comment:
That's a lot better. The difference looks to be in the dither, which is a judgement call for quantization. 

((you know, I bet the original was a truncation.)) 

Anyway, I think this is good enough.